conan/cli/commands/cache.py

```diff
@@ -54,22 +55,23 @@ def cache_path(conan_api: ConanAPI, parser, subparser, *args):
 @conan_subcommand()
 def cache_clean(conan_api: ConanAPI, parser, subparser, *args):
     """
-        Shows the path in the Conan cache af a given reference
+    Remove non-critical folders from the cache, like source, build and/or download
+    (.tgz store) ones.
     """
-    subparser.add_argument("pattern", help="selection pattern for references to clean")
+    subparser.add_argument("pattern", help="Selection pattern for references to clean")
     subparser.add_argument("-s", "--source", action='store_true', default=False,
                            help="Clean source folders")
     subparser.add_argument("-b", "--build", action='store_true', default=False,
-                           help="Clean source folders")
+                           help="Clean build folders")
     subparser.add_argument("-d", "--download", action='store_true', default=False,
                            help="Clean download folders")
     subparser.add_argument('-p', '--package-query', action=OnceArgument,
-                           help="Remove all packages (empty) or provide a query: "
+                           help="Remove only the packages matching a specific query, e.g., "
                                 "os=Windows AND (arch=x86 OR compiler=gcc)")
     args = parser.parse_args(*args)
 
     if not args.source and not args.build and not args.download:
-        raise ConanException("Define at least one argument from --source, --build, --download")
+        raise ConanException("Define at least one argument between [--source, --build, --download]")
```

Comment:
```suggestion
        raise ConanException("Define at least one argument among [--source, --build, --download]")
```

not sure about this, but I'd say between not great either

Reply:
Among, as between is for when discussing two items 👍